src/components/Sentence/index.jsx

```diff
@@ -1,9 +1,24 @@
-import React from 'react';
+import React, { useState, useEffect } from "react";
 
 export default function Sentence({ word, sentence }) {
+  const [wordTimer, setWordTimer] = useState(false);
+
+  useEffect(() => {
+    setWordTimer(false);
+    const timer = setTimeout(() => {
+      setWordTimer(true);
+    }, 1500);
+    return () => clearTimeout(timer);
+  }, [word]);
+
   return (
     <>
-      <h2>{word}</h2>
+      <h2
```

Comment:
Kind of an interesting problem that really won't likely be anything more than theoretical, but I think this should just be in a generic div tag, and should it be hidden from screen readers as well? I have to imagine a blind person learning to spell wouldn't likely be using our software but maybe? and in that case would not want the hidden word spelled out. If it was a blind parent teaching their child to spell, then they would definitely want the word available and not hidden from screen readers. 

I understand it's all theoretical and maybe we could do some research if we were interested. Until then, let's just make it a generic div tag.

Reply:
All the tags I've chosen are completely arbitrary, and did so just to see if it is functional. I will go back and change the tags once I start styling.

That's an interesting thought about screen readers. I wonder if screen readers can catch elements that are hidden with visibility css.